packages/devtools_app/lib/src/memory/flutter/memory_controller.dart

```diff
@@ -65,6 +67,55 @@ class MemoryController extends DisposableController
 
   String get memorySource => _memorySourceNotifier.value;
 
+  /// Starting chunk for slider based on pruneInterval.
+  double sliderValue = 1.0;
+
+  /// Number of interval chunks e.g., 5 minute interval has 3 chunks for 15 minutes of collected data
+  int numberOfChunks = 0;
+
+  /// Compute total timeline chunks used by Timeline slider.
+  int computeChunks() {
+    int chunks = 0;
+    if (memoryTimeline.data.isNotEmpty) {
+      final lastSampleTimestamp = memoryTimeline.data.last.timestamp.toDouble();
+      final firstSampleTimestamp =
+          memoryTimeline.data.first.timestamp.toDouble();
+      chunks = ((lastSampleTimestamp - firstSampleTimestamp) /
+              pruneIntervalDurationInMs)
```

Comment:
nit: rename to remove prune from name. I have found this to be confusing and think `interval` and `intervalDurationInMs` would be more intuitive.

Reply:
Done.